core/chains/evm/chain_set_test.go

```diff
@@ -24,28 +23,28 @@ func TestUpdateKeySpecificMaxGasPrice_NewEntry(t *testing.T) {
 	t.Parallel()
 
 	address := common.HexToAddress("0x1234567890")
-	price := big.NewInt(12345)
+	price := utils.NewWeiI(12345)
```

Comment:
Probs gonna rename this, call it `assets.NewWei` and rename `utils.NewWei` to `utils.IntToWei` or smth